src/services/tags/update-tag.test.js

```diff
@@ -0,0 +1,40 @@
+const {expect} = require('chai')
+const { faker } = require('@faker-js/faker')
+const updateTag = require("./update-tag")
+const {fixtures} = require('../../../test/utils')
+
+
+describe('Update Tag', () => {
+	
+	let tag
+	let updatedInfo 
+
+	before(async () => {
+		tag = await fixtures.insertTag() 
+		updatedInfo = {
+			tagId : tag.id,
+			title : faker.random.word(5),
+			content: faker.random.words()
+		}
+	})
+
+	it('should throw an error if tag does not exist', async () => 
+		
+		 expect(updateTag({updatedInfo, tagId: 230}))
+			.to.be.rejectedWith(
+				'Tag does not exist')
+	)
+
+	it('should update tag', async () => {
```

Comment:
this seems more like 'should return right response' than 'should update tag' 

a 'should update tag' test would like to know if the update was indeed made in the database

otherwise, if `updateTag` function just decides to return the arguments passed to it without an actual database update, the test will still pass

Reply:
i have made the necessary changes and pushed

Reply:
yea seen. I actually was implying you keep that test logic and name it` 'should return right response'` and then have another test case for the `'should update tag'`

The `'should update tag'`  should compare what is in the database with the `updatedInfo` which is generated, NOT with the response from the service function

the aim is to know if the update has actually been made. So the question we want to answer mostly is, if the update doesn't happen or the query to update the tag is removed, will the test still fail or pass? 

In this case, if I decide to change the implementation of the `updateTag` function to just make a `SELECT` query to return a tag without actually doing any update, it will be returned and be compared with the similar query in the test right now and they will be deemed as equal and the test will pass

Reply:
thank you very much for this explanation, I have made the change and pushed it to github